src/client/src/rt-components/adaptive-loader/AdaptiveLoader.tsx

```diff
@@ -62,16 +62,29 @@ const AdaptiveLoader: React.FC<Props> = React.memo(
     return (
       <svg width={sizeNum} height={sizeNum} data-qa="adaptive-loader__svg">
         {bars.map((item, i) => (
-          <Bar
-            type={type || 'primary'}
-            key={i}
-            height={barHeight}
-            width={barWidth}
-            x={extraWidth / 2 + i * (barWidth + seperationDistance)}
-            order={i}
-            moveDistance={moveDistance}
-            speed={speed || ANIMATION_SPEED}
-          />
+          // if speed is zero - show adaptive logo layout
+          speed === 0 ?
+            <Bar
+              type={type || 'primary'}
+              key={i}
+              height={barHeight}
+              width={barWidth}
+              x={extraWidth / 2 + i * (barWidth + seperationDistance)}
+              y={i === 0 ? 3 : i === 1 ? 4 : i === 2 ? 2 : 0}
```

Comment:
hardcoded coordinates of bars in Adaptive logo... this might be wrong is some cases, but I hope that people would not use speed===0 in lots of cases

Reply:
Should this not allow the speed to be `0` and have an <AdaptiveLogo /> as a separate component. Then you can do

```
loading ? <AdaptiveLoader speed={42} /> : <AdaptiveLogo />
```

Reply:
will revert to that, thanks (done that initially, but I didn't like the animation transition). It looks ok now. Thank you!

Reply:
done